LuminAi/src/main/frontend/src/services/PowerService.js

```diff
@@ -0,0 +1,26 @@
+function getUpdatedEnergyDevices() {
+
+    fetch('http://localhost:8080/api/devices/getData')
```

Comment:
Please make write the base url in the application properties of the quarkus project

Reply:
I will make this change tomorrow!